basics/basic-types.md

```diff
@@ -0,0 +1,124 @@
+# Grundläggande typer
+
+D erbjuder ett antal grundläggande typer som alltid har samma
+storlek **oavsett** vilken plattform. Det enda undantaget
+är typen `real` som erbjuder den högsta möjliga precision för
+flyttal. Det finns ingen skillnad mellan storleken på ett heltal
+oavsett om programmet är kompilerat för 32-bitar eller 64-bitars system.
+
+| typ                           | storlek
+|-------------------------------|------------
+|`bool`                         | 8-bit
+|`bool`, `ubyte`, `char`        | 8-bit
+|`short`, `ushort`, `wchar`     | 16-bit
+|`int`, `uint`, `dchar`         | 32-bit
+|`long`, `ulong`                | 32-bit
+
+#### Flyttalstyper:
+
+| typ     | storlek
+|---------|--------------------------------------------------
+|`float`  | 32-bit
+|`double` | 64-bit
+|`real`   | >= 64-bit (generellt 64-bit, men 80-bit på Intel x86 32-bit)
+
+Prefixet `u` betecknar en *osignerad* typ. `char` översätter till
+UTF-8 tecken, `wchar` används i UTF-16 strängar och `dchar`
+i UTF-32 strängar.
+
+En omvandling mellan variabler av olika typer är endast
+tillåtet av kompilatorn om ingen precision går förlorad. Dock,
+en konvertering mellan flyttalstyper (T.ex. `double` till `float`)
```

Comment:
Instead of starting the sentence with `Dock`. You could write:
`[...] (T.ex. double till float) är dock tillåtet.`
Or just remove the word altogether.